topostats/validation.py

```diff
@@ -595,6 +595,17 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
             ),
             "core_set": bool,
         },
+        "all_molecule_traces": {
+            "title": str,
+            "image_type": Or(
+                "binary",
+                "non-binary",
+                error=(
+                    "Invalid value in config 'coloured_boxes.image_type', valid values " "are 'binary' or 'non-binary'"
```

Comment:
Copy & pasta problem here, `error` needs tweaking to reflect its a problem with `all_molecule_traces`.